### DISCUSSION TITLE
1 out of 2049 subtasks failed. Please try again.

### DISCUSSION THREAD

**Discussion by petermarting** [mtalcott/google-photos-deduper#39] [Q&A] [answered]
Deduper had successfully gone through my nearly 200,000 photos in the "processing duplicates" phase and entered the subtask processing. About 5 hours in, it gave an error "1 out of 2049 subtasks failed. Please try again". I closed the terminal, logged out, and attempted to restart the process by running "docker-compose up" from the terminal again. I make it to the processing phase, but now the http://localhost/ page does not give any details about what is being processed and seems to continue indefinitely. I let it run overnight and nothing had changed. Any suggestion about how to pick up where I left off before or restart properly? Thanks!

**Comment by petermarting** [comment]
Just an update - by restarting my computer I was able to start over from scratch. I made it to the same point in the process but got the same error again.

**Reply by kzawadz** [reply, marked as answer]
@petermarting I found a solution to this. In /google-photos-deduper/app/lib/duplicate_image_detector.py add the following code after line 112 so it sits just after media_item = self.media_items[i]

`                if "storageFilename" not in media_item:
                    logging.warning(f"Missing 'storageFilename' in media_item: {media_item}")
                    continue  # Skip this item or handle it differently`
                    
Then rerun docker-compose up, and now all should be OK.

**Reply by petermarting** [reply]
That did it! Thanks so much!

**Reply by aleon1220** [reply]
> @petermarting I found a solution to this. In /google-photos-deduper/app/lib/duplicate_image_detector.py add the following code after line 112 so it sits just after media_item = self.media_items[i]
> 
> ` if "storageFilename" not in media_item: logging.warning(f"Missing 'storageFilename' in media_item: {media_item}") continue # Skip this item or handle it differently`
> 
> Then rerun docker-compose up, and now all should be OK.

is that code added to the main branch now @kzawadz ?
thanks a lot
